piksi_tools/console/settings_view.py

```diff
@@ -155,7 +155,7 @@ def __init__(self, name, section, value, ordering=0, settings=None):
                 self.readonly = True
         self._prevent_revert_thread = False
 
-    def revert_to_prior_value(self, name, old, new):
+    def revert_to_prior_value(self, name, old, new, error_value=1):
```

Comment:
Due to the behavior here:
https://github.com/swift-nav/piksi_tools/blob/1cbcd2c0e373b11aa1d2f0edfbede827895dd5e8/piksi_tools/console/settings_view.py#L211-L216
Let's create a timeout error condition and make that the default, which we can make -1 since that couldn't possibly come from an sbp message, which would use the `Unable to confirm that {0} was set to {1}.` verbiage. All other calls to this function should have an actual value from the device.